localflavor/bg/fields.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from .validators import egn_validator, eik_validator
+
+
+class EGNField(models.CharField):
```

Comment:
The naming of these fields was one of the hardest things. In Bulgaria:

-  ЕГН (transliterated EGN) stands for Единен Граждански Номер, which translated means Uniform Civil Number. 
- ЕИК (transliterated EIK) stands for Единен Идентификационен Код, which translated means Uniform Identity Code

The only way in English I saw ЕГН was in Wikipedia and it was just transliterated EGN. About ЕИК I never saw it written in English, and for that reason I just transliterated it also.

I can make the fields `BGUniformCivilNumber` and `BGUniformIdentityCode` if you prefer. I hope that someone will understand what are they because in the doc strings it is described.